draft-ietf-quic-recovery.md

```diff
@@ -1258,6 +1254,11 @@ kPersistentCongestionThreshold:
 Variables required to implement the congestion control mechanisms
 are described in this section.
 
+max_datagram_size:
+: The sender's current maximum payload size. Does not include UDP or IP
+  overhead.  The max datagram size is used for calculating initial and
+  minimum congestion windows. The RECOMMENDED value is 1200 bytes.
```

Comment:
This can't be less than 1200.  Worth adding?

Reply:
Done, PTAL.